### PR TITLE
Upstream Sync (License Change)

### PR DESCRIPTION
**Pull request by DEATHB4DEFEAT**
# Description

The license change!
All that *needs* reviewing here is the [README](https://github.com/Simple-Station/Parkstation-Friendly-Chainsaw/blob/upstream-sync-5/README.md), [LEGAL](https://github.com/Simple-Station/Parkstation-Friendly-Chainsaw/blob/upstream-sync-5/LEGAL.md), and [MARKERS](https://github.com/Simple-Station/Parkstation-Friendly-Chainsaw/blob/upstream-sync-5/MARKERS.md)

Update 4: I made my own markers file, look at the description history for the previous updates

### CODE REVIEW

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/medical.yml
Comment (Pspritechologist) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/medical.yml:11]: I don't believe this is an intentional change in this pr

**Review by Pspritechologist — CHANGES_REQUESTED** [review]
## MARKERS.md
- I still really think `// Parkstation-[TitleOfFeature]-Start - Optional clarification of why this change was done` should be `// Parkstation-[TitleOfFeature]-Start // Optional clarification of why this change was done`
- Ditto for YAML (`#`)
- File types listed at the bottom mention liscense, which isn't relevant for this file.
- "The following is the annotation formatting per language:"
## README.md
- Clarify that translations should go to EinsteinEngines, and not us. No need for us to maintain those anymore.
- "If you make any contributions, please make sure to read the markers section in MARKERS Any changes made to files belonging to our upstream should be properly marked in accordance to what is specified there." is missing punctuation after the word 'MARKERS'
- Add line "Before making a pr here, consider if the feature is specific to us, or would generally improve the codebase. More encompassing features should be made on our Upstream where plenty of the same Maintainers manage. If you're familiar with someone here, feel free to request a review from them in partiular.
- Python is *not* a build dependency, and technically speaking neither is Git.
- I would tell people to learn Git rather than run RUN_THIS.py, but that's me.
- I would say we put the Wizden launcher first. I gotta be honest, MV has been pretty unstable for me and overall has been worse to use.
- Standalone Wizden launcher listed but not MV 🧌
## LEGAL.MD
- Double check that EE is also doing the whole half-n-half in terms of licensing.

File: MARKERS.md
Comment (Pspritechologist) [inline MARKERS.md:50]:
```suggestion
[Your code here. This can be entirely original, or modified from the original file] # Parkstation-[TitleOfFeature] # Optional clarification of why this change was done
```

E

File: MARKERS.md
Comment (Pspritechologist) [inline MARKERS.md:58]:
```suggestion
# Parkstation-[TitleOfFeature]-Start # Optional clarification of why this change was done
```

E

File: README.md
Comment (Pspritechologist) [inline README.md:31]: This should be above in the section with contributing here, since it's not related to EE. Being broken up can be confusing.

File: README.md
Comment (Pspritechologist) [inline README.md:46]:
```suggestion
### Recommended Tools

> - Git (needed)
> - Python 3.7 or higher
> - Visual Studio Code (NOTE: NOT VISUAL STUDIO. Ask in Discord why.)
```

File: Resources/Prototypes/Catalog/VendingMachines/Inventories/medical.yml
Comment (Pspritechologist) [inline Resources/Prototypes/Catalog/VendingMachines/Inventories/medical.yml:1]: Don't think these are supposed to be here :P
I think these made their way into more than one pr??

File: Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml
Comment (Pspritechologist) [inline Resources/Prototypes/Entities/Clothing/Eyes/glasses.yml:1]: Don't think these are supposed to be here :P

**Review by Pspritechologist — CHANGES_REQUESTED** [review]

**Review by VMSolidus — APPROVED** [review]
This should be good to merge once Psprite's requested changes are made.

**Pull request merged**
